onnx_chainer/functions/normalization.py

```diff
@@ -10,34 +10,50 @@
 @support((1, 6, 7))
 def convert_BatchNormalization(func, opset_version, input_names,
                                output_names, context, parameters):
+    names = [context.get_name(v.get_variable().array) for v in func.inputs]
+    if names[1].startswith('param_'):
+        prefix = names[1][:-6]  # remove "_gamma"
```

Comment:
`[:-1-len('_gamma')]`?

Reply:
It seems that this part will be entirely changed in #185. I will keep this part as is in this PR.